rust-runtime/aws-smithy-wasm/Cargo.toml

```
@@ -0,0 +1,29 @@
+[package]
+name = "aws-smithy-wasm"
+version = "0.0.0-smithy-rs-head"
```

Our build/release system supports manual versioning of runtime crates now, and we'd like to move away from this auto-incrementing version string:
```suggestion
version = "0.1.0"
```